js/main.js

```diff
@@ -1,17 +1,94 @@
-const comment = document.querySelector('.social__footer-text');
+// Количество фото
+const AMOUNT_PHOTO = 25;
 
-function getRandomNumber(minNumber, maxNumber) {
-  if (maxNumber > minNumber && maxNumber > 0 && minNumber >= 0) {
-    minNumber = Math.ceil(minNumber);
-    maxNumber = Math.floor(maxNumber);
-    return Math.floor(Math.random() * (maxNumber - minNumber + 1)) + minNumber;
-  }
-  const error = 'Введите другие числа';
-  return error;
+// Количество лайков
+const MIN_AMOUNT_LIKE = 15;
+const MAX_AMOUNT_LIKE = 200;
+
+// Количество комментариев
+const AMOUNT_COMMENT = 3;
```

Comment:
```suggestion
const AMOUNT_COMMENTS = 3;
```

тут, я так понимаю, у тебя колличество комментариев - комментарии во мн ч